Project-1/README.md

```diff
@@ -67,9 +183,99 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
   (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
   (c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
   (d) ∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+  
+   (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  
+ (1) ¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+ (2) ¬∀y(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))
+ (3) ¬(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))
+ (4)¬(Pa ∧ Qa)
+ (5)¬(¬Pa ∨ ¬Qa)
+ (6).¬¬Pa
+ (7)¬¬Qa
+ (8)Qa
+ (9) Pa
+ (10) ¬Pa	(11) ¬Qa
+        x	       x
+ 
+  (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+  
+  (1) ¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  (2) ¬∀y(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))
+  (3) ¬(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))
+  (4) ¬(Pa ∧ Qa)
+  (5) ¬(¬Pa ∨ ¬Qa)
+  (6) ¬¬Pa
+  (7) ¬¬Qa
+  (8) Qa
+  (9) Pa
+  (10) ¬Pa	(11) ¬Qa
+        x	      x
+  
+  (c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+  
+  (1) ¬∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+  (2) ¬∀y((¬Pa ∨ ¬Qa) → ¬(Pa ∧ Qa))
+  (3) ¬((¬Pa ∨ ¬Qa) → ¬(Pa ∧ Qa))
+  (4) ¬Pa ∨ ¬Qa
+  (5) ¬¬(Pa ∧ Qa)
+  (6) Pa ∧ Qa
+  (7) Pa
+  (8) Qa
+  (9) ¬Pa	(10) ¬Qa
+     x		     x
+  
+  (d) ∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+  
+ (1) ¬∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+ (2) ¬∀y((¬Pa ∧ ¬Qa) → ¬(Pa ∨ Qa))
+ (3) ¬((¬Pa ∧ ¬Qa) → ¬(Pa ∨ Qa))
+ (4) ¬Pa ∧ ¬Qa
+ (5) ¬¬(Pa ∨ Qa)
+ (6) Pa ∨ Qa
+ (7) ¬Pa
+ (8) ¬Qa
+ (9) Pa		(10) Qa
+     x		     x
+     
 ```
 	
 9. Using a natural deduction proof generator - such as the one found here `https://proofs.openlogicproject.org/` - provide natural deduction proofs for each of De Morgan's laws. 
 
+¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+(1) ¬(Px ∧ Qx)		assumption
+(2) ¬(¬Px ∨ ¬Qx)	assumption
+(3) (Px ∧ Qx)		2, negated disjunction
+(4) ¬(Px ∧ Qx)		1,3 contradiction
+
+∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+(1) ¬(Px ∨ Qx)		assumption
+(2) (¬Px ∧ ¬Qx)		assumption
+(3) (Px ∨ Qx)		2, negated conjunction
+(4) ¬(Px ∨ Qx)		1,3 contradiction
+
+∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+(1) (¬Px ∨ ¬Qx)		assumption
+(2) ¬¬(Px ∧ Qx)		assumption
+(3) (Px ∧ Qx)		2, double negation
+(4) Px			3, conjunction
+(5) Qx			3, conjunction
+(6) (¬Px ∨ ¬Qx)		4,5,6 contradiction
```

Comment:
I don't understand what is happening in line 6

Reply:
∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
(1) (¬Px ∨ ¬Qx)		assumption
(2) ¬¬(Px ∧ Qx)		assumption
(3) (Px ∧ Qx)		2, double negation
(4) Px			3, conjunction
(5) Qx			3, conjunction
(6) (¬Px ∨ ¬Qx)		4,5,6 contradiction

I'm not sure either.  I copied and pasted the proof to try to analyze what's going on, but I used the natural deduction proof generator to find this answer so I'm not sure how to explain it.  

I'm not sure how the disjunction was created, but lines 4 and 5 have proved that Px and Qx are true (maybe?)  However, we have also already assumed (¬Px ∨ ¬Qx) to be true, thus revealing a contradiction if it is fair game to state (Px ∨ Qx) after proving Px and Qx.  Am I at least on track here?